lib/rig_inbound_gateway/api_proxy/handler/kafka.ex

```diff
@@ -279,7 +270,10 @@ defmodule RigInboundGateway.ApiProxy.Handler.Kafka do
         conn
         |> Tracing.Plug.put_resp_header(Tracing.context())
         |> Conn.put_resp_content_type("application/json")
-        |> Conn.send_resp(:ok, response)
+        |> Map.update!(:resp_headers, fn existing_headers ->
+          existing_headers ++ Map.to_list(response_headers)
+        end)
```

Comment:
Doesn't `existing_headers` likely already include a `Content-Type` header? (same comment in http.ex)